waterloop-site/src/state/blog/actions.ts

```diff
@@ -0,0 +1,9 @@
+import { Posts } from "blogs";
+import * as actionTypes from "../action-types";
+
+export const setPosts = (posts: Posts[]) => ({
+    type: actionTypes.BLOG_SET_POSTS,
+    payload: {
+        posts
+    }
+})
```

Comment:
Issue with BLOG_SET_POSTS
![image](https://user-images.githubusercontent.com/71536798/130693251-0df324c2-4caa-4f5f-9af7-96acea62005e.png)

This issue pertains in the other state controllers for blog.